xcpretty.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'rouge', '~> 1.8'
+
```

Comment:
@kylef if there's a hard dependency, then there's no need for the conditional :)
The good thing is that it has no dependencies, and that's something we valued a lot in `xcpretty` itself so far.

On the other hand - I think we should be moving towards homebrew because `xcpretty` should be treated like a command line tool rather than a gem.
I understand we can bundle that up together with rouge tho, just thinking out loud.

@kattrali what are your thoughts on this particular one?